GDAXSharp/Services/IEndPoint.cs

```diff
@@ -0,0 +1,28 @@
+﻿using System.Net.Http;
+
+namespace GDAXSharp.Services
+{
+    public interface IEndPoint
```

Comment:
i don't have an opinion on where this should live. what do you think?

Reply:
In Network? Or just where it is for now? Happy to be guided by your naming conventions...